src/createAnimatedComponent.js

```diff
@@ -216,6 +216,19 @@ export default function createAnimatedComponent(Component) {
     }
 
     _attachPropUpdater() {
+      if (Platform.OS === 'ios') {
+        for (const key in this.props) {
+          const value = this.props[key];
+          if (key === 'style' || key === 'animatedProps') {
+            const props = Array.isArray(value) ? value : [value];
+            props.forEach((prop) => {
+              if (prop && prop.viewName && !prop.viewName.value) {
+                prop.viewName.value = this._obtainViewName();
```

Comment:
can't we store viewName information along viewTag that is updated in `_attachAnimatedStyles` already?

Reply:
fixed c1d13d9